### PR TITLE
add a parameter for cert-manager version

### PR DESCRIPTION
**Pull request by jiaqiluo**
this PR allows us to pass in the cert-manager's version in automation

### CODE REVIEW

File: tests/validation/tests/v3_api/common.py [hunk 1]
```diff
@@ -196,6 +196,7 @@
 test_run_id = "test" + str(random.randint(10000, 99999))
 RANCHER_HOSTNAME_PREFIX = os.environ.get("RANCHER_HOSTNAME_PREFIX",
                                          test_run_id)
+CERT_MANAGER_VERSION = os.environ.get("RANCHER_CERT_MANAGER_VERSION", "v1.0.1")
```
Comment (sowmyav27) [inline tests/validation/tests/v3_api/common.py:199]: @jiaqiluo Is the default one going to be this `"v1.0.1"` or `0.15.0` ?

**Review by sangeethah — APPROVED** [review]
LGTM

File: tests/validation/tests/v3_api/common.py (hunk 1 above)
Comment (jiaqiluo) [reply tests/validation/tests/v3_api/common.py:199]: @sowmyav27 
1.0.1 will be supported in the coming rancher v2.4.9 and v2.5.0, so it should be ok for us to leave the default to be 1.0.1

we need to pass the value 0.15.0 for installing older version of Rancher

**Review by sowmyav27 — APPROVED** [review]
LGTM

**Pull request merged**